src/main/java/cinema/controller/AuthenticationController.java

```diff
@@ -0,0 +1,24 @@
+package cinema.controller;
+
+import cinema.model.dto.request.UserRequestDto;
+import cinema.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping
```

Comment:
what is the point of this annotation here?

Reply:
Fixed